src/addresses.js

```diff
@@ -1,15 +1,29 @@
 const AAVE_ADDRESSES_PROVIDER = {
-  '1': '0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5',
+  1: '0xB53C1a33016B2DC2fF3653530bfF1848a515c8c5',
 };
 const WETH = {
-  '1': '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+  1: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
 };
 const DAI = {
-  '1': '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+  1: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+};
+const xSUSHI = {
+  1: '0x8798249c2E607446EfB7Ad49eC89dD1865Ff4272',
+};
+const AAVE = {
+  1: '0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9',
+};
+const sUSD = {
+  1: '0x57ab1ec28d129707052df4df418d58a2d46d5f51',
+};
+
+// 8 decimals
+const wBTC = {
+  1: '0x2260fac5e5542a773aa44fbcfedf7c193bc2c599',
 };
 
 module.exports = {
   AAVE_ADDRESSES_PROVIDER,
   WETH,
-  DAI,
+  DAI: AAVE,
```

Comment:
this is the only relevant change - everything else is linting.
Do you have any idea on what could be wrong as DAI seems to be the only assets for which swapping works in a reliable way. Other 18 digit tokens like xSUSHI/AAVE/sUSD seem to throw most of the time.

Reply:
I don't understand why you're exporting AAVE as DAI, you can export the constants as they are and set a new constant to the appropriate token in the test.

Reply:
i thought changing the address would be enough to test this, didn't realize getRate et al used the hardcoded symbol :+1: